### ISSUE TITLE
[Issue] RS Bridge unstacks items being imported

### ISSUE THREAD

**Issue by iammael** [Seniorendi/AdvancedPeripherals#69]
**Describe**

When calling the method listItem() while items are getting imported in the system, the program while unstacks items in the RS disks.

**Reproduce**

1. Import any item in a constant flux
2. Call listItem() in a loop

**Screenshots/Videos**

https://user-images.githubusercontent.com/7931135/117329960-7a4a9380-ae95-11eb-876b-abbc67e70ea3.mp4

Code used:
```
rs = peripheral.find("rsBridge")


while true do
    rs.listItems()
    sleep(5)
end
```

**Versions:**

- Forge version: 36.1.2
- AdvancedPeripherals version: 0.5.3b

**Notes:**
Not tested with ME Bridge.

**Comment by Seniorendi** [comment]
Hey, could you try it with the latest RS and Forge version?

**Comment by iammael** [comment]
I'm still able to reproduce the bug using this modlist (should be latest for all) : 

- **Forge** : 36.1.16
- **Advanced Peripherals**: 0.5.3b
- **CC:Tweaked** : 1.16.4-1.95.3
- **Refined Storage**: 1.9.12
- JEI
- Storage Drawers (used for importing items. Do note that I don't use drawers at all in my previous setup)

https://user-images.githubusercontent.com/7931135/117379985-04681b80-aed9-11eb-8c60-360ee71bbd25.mp4

**Comment by Seniorendi** [comment]
[advancedperipherals-0.5.4b.zip](https://github.com/Seniorendi/AdvancedPeripherals/files/6440427/advancedperipherals-0.5.4b.zip)
Could you try if this version fixes your problem? Should also fix #68

**Comment by iammael** [comment]
I can still reproduce both this issue and #68 with the version you just provided.

**Comment by Seniorendi** [comment]
I found a solution, I can call `invalidate` for the cache but that spams a message every time you use listItems.
I'll try to find a better solution.
[advancedperipherals-0.5.4b.zip](https://github.com/Seniorendi/AdvancedPeripherals/files/6445598/advancedperipherals-0.5.4b.zip)
But you can try it, if you want.
Unzip it before using.

**Comment by iammael** [comment]
Okay this seems to work as expected. I didn't even see the spam message (even in the log, but I didn't activate debug during testing).

I'll let you decide whether or not closing this issue.

**Comment by Seniorendi** [comment]
What you di

**Comment by Seniorendi** [comment]
What you did not seen the message from refined storage?

**Comment by Seniorendi** [comment]
Sorry, I didn't want to close it

**Comment by Seniorendi** [comment]
Fixed in 0.5.4b
